spring/src/test/java/org/jdbi/v3/spring/TestJdbiFactoryBean.java

```diff
@@ -72,7 +70,7 @@ public void testFailsViaException() throws Exception
             });
         }
         catch (ForceRollback e) {
-            assertTrue(true);
+            assertThat(e).isNotNull();
```

Comment:
That's a nice observation. I used this pattern and also moved several checks to the `@Test(expect)` declaration, where it's possible.